scripts/epoch.bat

```diff
@@ -0,0 +1,5 @@
+SETLOCAL ENABLEEXTENSIONS
+
+SET REL_NAME=aeternity
+
+call %~dp0/%REL_NAME% %*
```

Comment:
Could you please confirm that [this test](https://github.com/aeternity/aeternity/pull/2022#issuecomment-454965501) passes.

Reply:
I will report back until tomorrow on the result.